app/views/merchant_dashboards/index.html.erb

```diff
@@ -0,0 +1,26 @@
+<h1>Merchant Dashboard</h1>
+
+<%= @merchant.name %>
+
+<%= link_to "My Items", "/merchants/#{@merchant.id}/items" %>
+<%= link_to "My Invoices", "/merchants/#{@merchant.id}/invoices" %>
+
+<div id='fav-customers'>
+<h4>Favorite Customers</h4>
+<ol>
+<% @merchant.favorite_customers.each do |customer| %>
+  <li><%= customer.first_name %> <%= customer.last_name %> - <%= customer.successful_transactions(@merchant.id) %> purchases</li>
+<% end %>
+</ol>
+</div>
+
+
+<div id='items-ready-to-ship'>
+<h4>Items Ready to Ship</h4>
+<% @merchant.items_ready_to_ship.each do |data| %>
+  <p>
+  <%= data.item_name %> - Invoice #<%= link_to "#{data.invoice_id}", "/merchants/#{@merchant.id}/invoices/#{data.invoice_id}" %> - <%= data.invoice_created_at.strftime("%A, %B %-d, %Y") %>
+  </p>
+<% end %>
```

Comment:
What is data referring to here, is it an invoice item?

Reply:
its a customer AR object we created in the method - similar to yours it's not referring to a single object so I wasn't sure what to call it. data has the name of an item, the invoice id for that item, and the invoice created_at